packages/wdio/__tests__/wdio.test.ts

```diff
@@ -11,6 +11,8 @@ import { htmlFileWithA11yIssues, htmlFileWithNoA11yIssues } from '@sa11y/test-ut
 import { A11yError } from '@sa11y/format';
 import { axeRuntimeExceptionMsgPrefix } from '@sa11y/common';
 
+// TODO (chore): Raise issue with WebdriverIO - 'sync' missing 'default' in ts def
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-var-requires
 const sync = require('@wdio/sync').default;
```

Comment:
What about this?
```
import sync from '@wdio/sync';
```

Reply:
I tried that. It results in the tests being skipped with wdio runner reporting that 1 suite passed without running any tests weirdly and also the following error.

```
TS1259: Module '"@wdio/sync"' can only be default-imported using the 'esModuleInterop' flag  webdriverio.d.ts(46, 5): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```

I do have the `"esModuleInterop": true,` in tsconfig.

Also tried `import sync = require('@wdio/sync');` as suggested in the [TypeScript: Handbook - Modules](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require)
> When exporting a module using export =, TypeScript-specific import module = require("module") must be used to import the module.

Even though typescript is happy with that, the tests continue to be skipped.

Think this could be related to the fact that [ts-node](https://github.com/TypeStrong/ts-node) is used by wdio.conf.js and it might be interpreting the tsconfig differently? I tried reusing babel (which is used by jest) instead of ts-node but couldn't get it working (https://github.com/webdriverio/webdriverio/issues/5588)

Please let me know if you have any suggestions.

Reply:
Added a TODO to reflect this for now

Reply:
Hm weird... Yeah that's fine.